spec/factories.rb

```diff
@@ -16,5 +16,6 @@
   factory :user do
     sequence(:name) { |n| "user name#{n}" }
     sequence(:email) { |n| "user email#{n}" }
+    password_digest "password"
```

Comment:
I believe this should just be `password "password"` since the `has_secure_password` deals with the converting it to a digest

Reply:
Thanks Elle will try that out